### PR TITLE
Add simple variant for tabs

### PR DESCRIPTION
**Pull request by mfrachet**
## Description

Add simple variant for the tabs component

## Demo

Example on : https://design-system-git-tabs-simple-strapijs.vercel.app/?path=/story/design-system-molecules-tabs--simple

## API

```diff
<TabGroup
   label="Some stuff for the label"
+  variant="simple"
>
  <Tabs>
    <Tab>First</Tab>
    <Tab>Second</Tab>
    <Tab>Third</Tab>
  </Tabs>
  <TabPanels>
    <TabPanel>First panel</TabPanel>
    <TabPanel>Second panel</TabPanel>
    <TabPanel>Third panel</TabPanel>
  </TabPanels>
</TabGroup>
```

## Voiceover

Tabs using the 'simple' variant

### CODE REVIEW

**Comment by ronronscelestes** [comment]
Neat, LGTM

**Review by ronronscelestes — COMMENTED** [review]
Neat, LGTM

File: packages/strapi-parts/src/Tabs/components.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import { Box } from '../Box';
+import { Row } from '../Row';
+
+/** Simple variant */
+export const SimpleTabBox = styled(Box)`
+  border-bottom: 2px solid
+    ${({ theme, selected, hasError }) => {
+      if (selected) {
+        if (hasError) {
+          return theme.colors.danger600;
```
Comment (HichamELBSI) [inline packages/strapi-parts/src/Tabs/components.js:11]: The error color is displayed only if the tab is selected?

**Review by maevalienard — APPROVED** [review]

File: packages/strapi-parts/src/Tabs/components.js (hunk 1 above)
Comment (mfrachet) [reply packages/strapi-parts/src/Tabs/components.js:11]: The text color, in error, is always red. However, the border should be red and visible only when the tab is selected, according to figma 😊

**Pull request merged**